packages/mdl-radio/event-bus.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Class to coordinate between between components without strict dependencies.
+ */
+export default class EventBus {
```

Comment:
I think we'd need some javascript oriented coordination mechanism, otherwise the checked property would get out of sync if another button was checked. The change event for radio inputs only fires for the particular item that is checked, not the other ones, so they don't know that they're unchecked from the JS side.

The only alternative to the event bus that I thought of would require an explicit dependency on a radio group class being instantiated and passed to the radio buttons; it seemed too cumbersome to me.

I'll switch to the EventEmitter unless there's another way.